### PR TITLE
feat: include minimum software reqs in faq

### PR DESCRIPTION
**Pull request by jithine**

### CODE REVIEW

**Review by vengopal — APPROVED** [review]

File: docs/user-guide/FAQ.md [hunk 1]
```diff
@@ -137,3 +139,11 @@ You can set the environment variable [`GIT_SHALLOW_CLONE`](./environment-variabl
 By default Screwdriver will shallow clone source repositories to a depth of 50. Screwdriver will also enable the `--no-single-branch` flag by default.
 
 If shallow cloning is left enabled and you wish to push back to your git repository, your image must contain a git version of 1.9 or later. Alternatively, you may use the version of git bundled with Screwdriver by invoking `sd-step exec core/git "GIT COMMAND"`.
+
+## What are the minimum software requirements for a build image?
+
+Screwdriver has no restriction on build container image. However it should have at the minimum `curl` && `openssh` installed.
+
+Also if `image` is Apline based then an extra workaround is required in the form of the following symlink. `mkdir -p /lib64 && ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2`
```
Comment (tkyi) [inline docs/user-guide/FAQ.md]:
```suggestion
Also, if the `image` is Alpine-based, an extra workaround is required in the form of the following symlink: `mkdir -p /lib64 && ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2`
```

File: docs/user-guide/FAQ.md [hunk 2]
```diff
@@ -137,3 +139,11 @@ You can set the environment variable [`GIT_SHALLOW_CLONE`](./environment-variabl
...
+
+Alternatively we can also try setting `LD_LIBRARY_PATH` to include musl library paths.
```
Comment (tkyi) [inline docs/user-guide/FAQ.md]: is this line for the Screwdriver team or for the user?
if it's something the user can try, I think we should change `we` to `you`

File: docs/user-guide/FAQ.md [hunk 3]
```diff
@@ -37,6 +37,8 @@ toc:
       url: "#how-do-i-speed-up-time-to-upload-artifacts"
     - title: How do I disable shallow cloning?
       url: "#how-do-i-disable-shallow-cloning"
+    - title: What are the minimum soft requirements for a build image?
```
Comment (tkyi) [inline docs/user-guide/FAQ.md]:
```suggestion
    - title: What are the minimum software requirements for a build image?
```

File: docs/user-guide/FAQ.md [hunk 4]
```diff
@@ -137,3 +139,11 @@ You can set the environment variable [`GIT_SHALLOW_CLONE`](./environment-variabl
 By default Screwdriver will shallow clone source repositories to a depth of 50. Screwdriver will also enable the `--no-single-branch` flag by default.
 
 If shallow cloning is left enabled and you wish to push back to your git repository, your image must contain a git version of 1.9 or later. Alternatively, you may use the version of git bundled with Screwdriver by invoking `sd-step exec core/git "GIT COMMAND"`.
+
+## What are the minimum software requirements for a build image?
+
+Screwdriver has no restriction on build container image. However it should have at the minimum `curl` && `openssh` installed.
+
+Also, if the `image` is Apline-based, an extra workaround is required in the form of the following symlink. `mkdir -p /lib64 && ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2`
```
Comment (tkyi) [inline docs/user-guide/FAQ.md]: still have a typo with `Alpine`

```suggestion
Also, if the `image` is Alpine-based, an extra workaround is required in the form of the following symlink: `mkdir -p /lib64 && ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2`
```

Comment (jithine) [reply docs/user-guide/FAQ.md]: oops fixed.

**Review by chestery — APPROVED** [review]

**Pull request merged**